src/lib/_common.scss

```diff
@@ -117,34 +118,51 @@ $header-height: 48px;
     }
 }
 
-.bp {
-    align-items: center;
-    background-color: $ffive;
-    border: 0 none;
+.bp-body {
     bottom: 0;
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
     left: 0;
-    margin: 0;
-    outline: none;
-    -webkit-overflow-scrolling: touch;
-    padding: 0;
     position: absolute;
     right: 0;
     top: 0;
 
-    // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
-    &.bp-fullscreen-unsupported {
+    .bp-thumbnails-container {
+        background-color: $ffive;
+        border-right: solid 1px $seesee;
+        width: 180px;
+    }
+
+    .bp {
+        align-items: center;
+        background-color: $ffive;
+        border: 0 none;
         bottom: 0;
+        display: flex;
+        flex: 1;
+        flex-direction: column;
+        justify-content: center;
         left: 0;
-        position: fixed;
+        margin: 0;
+        outline: none;
+        -webkit-overflow-scrolling: touch;
+        padding: 0;
+        position: relative;
         right: 0;
         top: 0;
-    }
 
-    &.bp-dark {
-        background-color: $black;
+        // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
+        &.bp-fullscreen-unsupported {
+            bottom: 0;
+            left: 0;
+            position: fixed;
+            right: 0;
+            top: 0;
+        }
+
+        &.bp-dark {
```

Comment:
Seems like this should still live on the root container.

Reply:
Actually it seems like it's only applied for the video player, in which case I think we can leave it as is

Reply:
I think setting the option `header` to `dark` will also trigger this ```String value of none or dark or light that controls header visibility and theme```.

Reply:
yeah I tried that but didn't see this css class applied, only when using the video player